.github/actions/build/action.yml

```diff
@@ -10,10 +10,11 @@ runs:
   steps:
     - name: Build distribution files
       shell: bash
+      working-directory: packages/sdk/server-ai
```

Comment:
There are a couple options here. The js-core repository passes in the workspace directory and has individual CI build workflows for each project (https://github.com/launchdarkly/js-core/blob/main/.github/workflows/release-please.yml).

The dotnet repository uses shared workflows but sets a github_actions.env file for each package that is loaded during the CI / build process https://github.com/launchdarkly/dotnet-core/tree/main/pkgs/sdk/server-ai.